collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+
+# If the cluster version is < 4.5 skip this test
+VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}')
+COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)
+if [[ ${COMPARE} != "4.5" ]]; then
+  exit 0
+fi
+
+KUBE_API_SERVERS=$(oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{range .items[*]}{.metadata.name}{" "}{.status.podIP}{" "}{range .spec.containers[?(.name=="kube-apiserver")]}{.ports[0].hostPort}{"\n"}')
```

Comment:
nit: could you add a comment that would show what we would get from running this command?